tests/node/app-boot-test.js

```diff
@@ -8,123 +8,117 @@ function assertHTMLMatches(assert, actualHTML, expectedHTML) {
 
 appModule("App Boot");
 
-// Because running these tests relies on certain feature flags being
-// enabled, we check to see if those flags are on before registering
-// the tests. Once all feature flags are on by default we can remove
-// this conditional.
-if (appModule.canRunTests) {
```

Comment:
I guess so. I'm not sure of the actual process of when things get removed. /cc @rwjblue